Project/contracts/CustomBallot.sol

```diff
@@ -83,13 +85,15 @@ contract CustomBallot {
     }
 
     /// @notice Get the voting power of the sender address
+    /// @return votingPower_ voting power
```

Comment:
Please add annotation for all return values

Reply:
But they are added!

Reply:
oh!  didn't see that.  you're good to go. sorry!